City-Bureau/city-scrapers-akr
# 🕷️ Fix spider: Akron-Summit County Public Library

## What's this PR do?

Fixes our Akron-Summit County Public Library spider (aka. `summ_library`).

## Why are we doing this?

The spider broke as intended. The default start time, 4pm, had shift to 4:30pm. However, while fixing the default time it was discovered that the scraper was no longer properly scraping the minutes for each meeting. Because this is a relatively minor agency with few meetings, I decided to drop this behavior in order to simplify the spider and make it more maintainable.

## Steps to manually test

After installing the project using `pipenv`:

1. Activate the virtual environment:
```
pipenv shell
```
2. Run the spider:
```
scrapy crawl summ_library -O test_output.csv
```
3. Monitor the stdout and ensure that the crawl proceeds without raising any errors. Pay attention to the final status report from scrapy.

4. Inspect `test_output.csv` to ensure the data looks valid. I suggest opening a few of the URLs under the source column of test_output.csv and comparing the data for the row with what you see on the page.

## Are there any smells or added technical debt to note?
